message/validation/validation_test.go

```diff
@@ -1383,22 +1383,6 @@ func Test_ValidateSSVMessage(t *testing.T) {
 		require.ErrorIs(t, err, ErrEventMessage)
 	})
 
-	// Receive a dkg message from an operator that is not myself should receive an error
-	t.Run("dkg message", func(t *testing.T) {
-		validator := New(netCfg, validatorStore, dutyStore, signatureVerifier).(*messageValidator)
-
-		slot := netCfg.Beacon.FirstSlotAtEpoch(1)
-
-		signedSSVMessage := generateSignedMessage(ks, committeeIdentifier, slot)
-		signedSSVMessage.SSVMessage.MsgType = spectypes.DKGMsgType
-
-		receivedAt := netCfg.Beacon.GetSlotStartTime(slot)
-		topicID := commons.CommitteeTopicID(spectypes.CommitteeID(signedSSVMessage.SSVMessage.GetID().GetDutyExecutorID()[16:]))[0]
-
-		_, err = validator.handleSignedSSVMessage(signedSSVMessage, topicID, receivedAt)
-		require.ErrorIs(t, err, ErrDKGMessage)
-	})
-
```

Comment:
obviously this should be removed

however maybe we should re-add this test with another non-supported msg type like you did here: https://github.com/ssvlabs/ssv/pull/1728/files#diff-ca2de5e8e7ca8dc378f4db4521e44e40bfcec11da025a6719a25487003843338R313

just to preserve this kind of test in the code

Reply:
Added a similar test